apps/chat/src/components/Chat/ChatInput/ChatInputMessage.tsx

```diff
@@ -475,6 +494,7 @@ export const ChatInputMessage = ({
             onClose={() => setIsModalVisible(false)}
           />
         )}
+        <ReplyVariables />
```

Comment:
I think typo is here
```suggestion
        <ReplayVariables />
```